app/code/community/Oyst/OneClick/etc/system.xml

```diff
@@ -616,6 +564,28 @@
                         </time>
                     </fields>
                 </order_delay>
+
+                <!-- button wrapper styles -->
+                <button_wrapper_styles translate="label">
+                    <label>Add custom styles to button wrapper of product page</label>
+                    <frontend_type>text</frontend_type>
+                    <sort_order>70</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <fields>
+                        <styles translate="label comment">
```

Comment:
After rebase from master, keep only the field `styles` and add it in `customization_button_product_page`

Reply:
@rus-daniel please move the field `styles` in the group `customization_button_product_page` and remove the groupe you created